src/main/java/com/exasol/containers/VersionBasedExasolDockerImageReference.java

```diff
@@ -14,41 +14,32 @@
  * </p>
  */
 class VersionBasedExasolDockerImageReference implements ExasolDockerImageReference {
-    private static final int VERSION_NOT_PRESENT = -1;
+    private static final int DEFAULT_DOCKER_IMAGE_REVISION = 1;
+    public static final int VERSION_NOT_PRESENT = -1;
+    public static final String SUFFIX_NOT_PRESENT = null;
     private final int major;
     private final int minor;
     private final int fix;
     private final int dockerImageRevision;
+    private final String suffix;
 
     /**
      * Create a docker image reference from a fully specified version.
      *
      * @param major               major database version
      * @param minor               minor database version
      * @param fix                 fix version (aka. "patch level")
+     * @param suffix              suffix like "RC1" or "beta"
      * @param dockerImageRevision revision number of the docker image
      */
-    public VersionBasedExasolDockerImageReference(final int major, final int minor, final int fix,
+    public VersionBasedExasolDockerImageReference(final int major, final int minor, final int fix, final String suffix,
             final int dockerImageRevision) {
         this.major = major;
         this.minor = minor;
         this.fix = fix;
-        this.dockerImageRevision = dockerImageRevision;
-    }
-
-    /**
-     * Create a version omitting the optional Docker image revision.
-     * <p>
-     * Since for {@code docker-db} versions below 7 the image revision was mandatory, the revision automatically
-     * defaults to 1.
-     * </p>
-     *
-     * @param major major database version
-     * @param minor minor database version
-     * @param fix   fix version (aka. "patch level")
-     */
-    public VersionBasedExasolDockerImageReference(final int major, final int minor, final int fix) {
-        this(major, minor, fix, (major < 7) ? 1 : VERSION_NOT_PRESENT);
+        this.suffix = suffix;
+        this.dockerImageRevision = ((dockerImageRevision == VERSION_NOT_PRESENT) && !isExasolSevenOrLater()) ? 1
+                : dockerImageRevision;
```

Comment:
Can this `1` on true condition for ternary op be changed to `DEFAULT_DOCKER_REVISION`? I was not sure if they are same.

Reply:
Fixed.